beancards/src/main/java/com/bbdgrads/beancards/Entities/Player.java

```diff
@@ -8,43 +8,43 @@
 import jakarta.persistence.Table;
 
 @Entity
-@Table(name = "players")
```

Comment:
Its not important whether we use Camel or Pascal case.. We have to be consistent so just change all tables to whatever I chose to go with Camel but if you want to change just do it for all tables